docs/abi.md

```diff
@@ -237,11 +237,13 @@ If creating the specified Node would violate
 [information flow control](/docs/concepts.md#labels), returns
 `ERR_PERMISSION_DENIED`.
 
-- `param[0]: usize`: Source buffer holding serialized `NodeConfiguration`
-- `param[1]: usize`: Serialized NodeConfiguration size in bytes
-- `param[2]: usize`: Source buffer holding serialized `Label`
-- `param[3]: usize`: Label size in bytes
-- `param[4]: usize`: Handle to channel
+- `param[0]: usize`: Source buffer holding the UTF-8 encoded name
+- `param[1]: usize`: Name size in bytes
+- `param[2]: usize`: Source buffer holding serialized `NodeConfiguration`
+- `param[3]: usize`: Serialized NodeConfiguration size in bytes
+- `param[4]: usize`: Source buffer holding serialized `Label`
+- `param[5]: usize`: Label size in bytes
+- `param[6]: usize`: Handle to channel
```

Comment:
Is it possible to leak data through node names?
Who can read these names?
Should we consider disallowing Oak applications from reading node names, or apply IFC to them too?

Reply:
The runtime can read node names, and they are displayed in the introspection graph. Node names cannot be changed after creation. At the moment we only allow public nodes to create other nodes, so it it not possible for node names to be based on non-public information.